paasta_tools/cli/cmds/logs.py

```diff
@@ -100,9 +103,37 @@ def add_subparser(subparsers):
         default=DEFAULT_SOA_DIR,
         help="define a different soa config directory",
     )
+
+    status_parser.add_argument(
+        '-fr', '--from', dest='time_from',
```

Comment:
Usually, single dash (-) arguments have single-character names. I'm not sure whether argparse supports having `-fr`, but even if it does, we should probably stick with the unix standard.